magento/ece-tools
# MAGECLOUD-4419: Add error details for failed functional tests in Travis

### Description

Display error message and stack trace for failed functional tests on Travis.

### Fixed Issues (if relevant)

https://magento2.atlassian.net/browse/MAGECLOUD-4419

### Manual testing scenarios

1. Successful build on Travis.
2. If build fails in Travis due to failed functional tests, the build output should include the error message and stack trace for the failed test.

### Contribution checklist
